src/test/kotlin/day03/HeroTest.kt

```
@@ -0,0 +1,30 @@
+    @org.junit.Test
+    fun sleep() {
+        //준비
+        val hero = Hero("홍길동", 10)
+        //실행
+        hero.sleep()
+        assertEquals(100, hero.hp)
+    }
+
+    @org.junit.Test
+    fun run() {
+    }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)